src/tests/ftest/pool/create_all_vm.py

```diff
@@ -22,7 +22,14 @@ def setUp(self):
         self.scm_avail_bytes = self.get_available_bytes()
 
     def get_available_bytes(self, ranks=None):
-        """Return the available size of SCM storage."""
+        """Return the available size of SCM storage.
+
+        Args:
+            ranks (list): List of server ranks to filter. Defaults to None.
```

Comment:
From reading the code we typically do the following for optional args which this appears to be.  Not worth re-pushing for however.
```suggestion
            ranks (list, optional): List of server ranks to filter. Defaults to None.
```

Reply:
> From reading the code we typically do the following for optional args which this appears to be. Not worth re-pushing for however.

No problem to push it again :)

- [ ] Fix docstring